wheel/README.md

```diff
@@ -1 +1,12 @@
 The `chia_rs` wheel contains python bindings for code from the `chia` crate.
+
+To run the tests:
+```
+cd wheel
+python -m venv venv
+. ./venv/bin/activate
+python -m pip install -r requirements.txt
+maturin develop
+python -m pytest ../tests
+```
+
```

Comment:
Now, when you're doing development, you probably *do* want `maturin` in your local venv so you can test rust changes by doing `maturin develop` rather than `pip install -e .` again, which takes longer because it creates a sandboxed venv to do the build.

So to do development, I guess ideally what you want

```
cd wheel
python -m venv venv
. ./venv/bin/activate
python -m pip install maturin==(whatever the version listed in pyproject.toml is)
python -m pip install -e '.[dev]'
maturin develop
python -m pytest ../tests
```

The `.[dev]` line means "please install additional package that this `pyproject.toml` file declares as being useful for development", so stuff like `pytest`, `ruff`, `mypy`, etc. We have to add stuff like in https://github.com/Chia-Network/hsms/blob/main/pyproject.toml#L19

Also, for obvious reasons, I'm not happy with the `python -m pip install maturin==(whatever the version listed in pyproject.toml is)` line. We could add `maturin` to the `dev = ["maturin==...]` line, but then we would have it in two places (the build section and the `dev` section) which is a recipe for disaster if they get out of sync and some incompatibility pops up and it works locally because we're using the `dev` version of `maturin` but not in CI because it uses the `build` version. I expect there is some `pip` thing to install the `build` tools, but I don't know what it is.

This comment is very dense with information, so feel free to ask for clarification if I'm not being clear.

Reply:
the python dependencies are all `'.[dev]'` dependencies as the python is only used for running pytests

Reply:
There is a `pyproject.toml` file so there is a “python” project… it’s the wheel `chia_rs`. It happens to not have any python source code at the moment, but that’s okay. The CI still uses the `pyproject.toml` file to build it (or at least, it should!). The idea of `pyproject.toml` is to standardize how python-targeted wheels build lifecycles are specified, so IMO it's reasonable to add the dev dependencies here even if most developers end up using the `maturin develop` shortcut during their build/test development cycle.

Reply:
I guess you're talking about https://github.com/Chia-Network/chia_rs/blob/main/README.md#python-tests ?

It might be helpful to note in the README that this mechanism to run the tests is a hack that is necessary because the `chia_rs` and `chia_blockchain` wheels currently have a cyclic dependency. At some point this can be fixed and then we'll be able to run the `chia_rs` tests with `pip install .[tests]` and `pyproject.toml` like I was hinting at above.

In any case, I think we can close this PR without merging it now. Thanks to @matt-o-how for bringing up this discussion. Prior to this, I hadn't internalized the cyclic dependency with `chia-blockchain`.